dev/reproducibility.py

```diff
@@ -0,0 +1,95 @@
+import argparse
+import json
+import os
+import shutil
+import nibabel as nib
+
+import numpy as np
+import pandas as pd
+
+from ivadomed import main as ivado
+
+
+def get_parser():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-c", "--config", required=True, nargs="+", help="Base config file path.")
+    parser.add_argument("-n", "--iterations", default=10, type=int, help="Number of Monte Carlo iterations.")
+    parser.add_argument("-o", "--output-path", nargs="+", dest="output_path", required=True,
+                        type=str, help="Output directory name without extention to save final csv file. There should "
+                                       "be the same number of output files parameters as the number of config files.")
+    return parser
+
+
+def get_results(context):
+    context["command"] = "eval"
+    pred_mask_path = os.path.join(context["log_directory"], "pred_masks")
+    if os.path.exists(pred_mask_path):
+        shutil.rmtree(pred_mask_path)
+
+    # RandomAffine will be applied during testing
+    if "dataset_type" in context["transformation"]["RandomAffine"]:
+        del context["transformation"]["RandomAffine"]["dataset_type"]
```

Comment:
That's just a safety thing to ensure this is run at eval time, right? which is normally not the case so I understand why needed here.

Reply:
Exact, to avoid a config file where RandomAffine would only be applied during training (which is usually the case)